test/underlying_timeout.cpp

```diff
@@ -261,10 +277,20 @@ BOOST_AUTO_TEST_CASE( connect_tls_ashs ) {
     std::size_t pos = path.find_last_of("/\\");
     std::string base = (pos == std::string::npos) ? "" : path.substr(0, pos + 1);
 
-    as::ssl::context ctx {as::ssl::context::tlsv12};
+    tls::context ctx {tls::context::tlsv12};
+
+#if defined(MQTT_USE_TLS)
+#if defined(MQTT_USE_GNU_TLS)
+    ctx.use_verify_file(base + "cacert.pem", tls::context::pem);
```

Comment:
"load_verify_file" is used in enough places that i'd rather see a single function that takes the context by reference, and the path to the file to use, and abstracts which member function to call, than see the #if's in a bunch of different places.

Reply:
Yeah, I can do that.

Reply:
Now that I've switched to the same API name load_verify_file, I'll skip creating a function to minimize the change-set